tools/downloader/topologies_test.py

```diff
@@ -0,0 +1,75 @@
+import unittest
```

Comment:
I think there is no need in special tests, since it could be done in demo [testing](https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/tests/run_tests.py). @IRDonch, @Wovchena are you agree?

Reply:
I don't agree. Demo tests are pretty superficial. They can only test whatever usages of the API the demos contain, and can only check whether the demo crashed or not, not any fine-grained checks. You get better coverage with dedicated tests.